packages/react-scripts/config/webpack.config.js

```diff
@@ -410,7 +410,7 @@ module.exports = function(webpackEnv) {
             },
             // load locale files
             {
-              test: /locales/,
+              test: /locales.index\.js$/,
```

Comment:
I don't understand this, you're using the `.` to represent the `/`? Is there some other value we would ever want to allow there?

Reply:
Yeah, it's just to represent the `/`, can we assume it will be `/` and not `\`?

Reply:
webpack always uses normalized paths for evaluation I believe, so never \